### DISCUSSION TITLE
Equivalent to --verbose --verbose for env var

### DISCUSSION THREAD

**Discussion by OscarVanL** [abraunegg/onedrive#2554] [Q&A]
Hi,

I currently launch onedrive-client via a container with environment variables to control the config like this:
https://github.com/abraunegg/onedrive/blob/1a88d33be3e2c6747c5621ab9d97df9cd3ee727a/docs/Docker.md?plain=1#L291-L294

Unfortunately I am having some errors that have suddenly started in production with no apparent cause, so I would like to debug this further.

I have set `ONEDRIVE_VERBOSE=1`, but still these errors are not quite detailed enough for me to figure out the problem.

I saw in some issues it has been suggested to use `--verbose --verbose` (providing the flag twice to put it into a "super verbose" mode), but am not sure what the equivalent is for environment variable configuration.

I tried setting `ONEDRIVE_VERBOSE=2`, but it looks to have made no difference to the verbosity of the logs, so I assume this is not the correct way to achieve this.

Any pointers would be greatly appreciated. Thank you.

**Comment by OscarVanL** [comment]
Nevermind, I RTFM a bit harder and found [this table](https://github.com/abraunegg/onedrive/blob/1a88d33be3e2c6747c5621ab9d97df9cd3ee727a/docs/Docker.md#supported-docker-environment-variables) which has exactly what I need: `ONEDRIVE_DEBUG`. Thank you for writing great detailed docs, and apologies for not reading them better.

**Reply by abraunegg** [reply]
No problem - glad you found what you were looking for
